client/state/selectors/test/get-sites.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { getSites } from '../';
+
+describe( 'getSites()', () => {
+	it( 'should return an empty array if no sites in state', () => {
+		const state = {
+			sites: {
+				items: {}
+			}
+		};
+		const sites = getSites( state );
+		expect( sites ).to.eql( [] );
+	} );
+
+	it( 'should return all the sites in state', () => {
+		const state = {
+			sites: {
+				items: {
+					2916284: {
+						ID: 2916284,
+						visible: true,
+						name: 'WordPress.com Example Blog',
+						URL: 'https://example.wordpress.com',
+						options: {
+							unmapped_url: 'https://example.wordpress.com'
+						}
+					},
+					2916285: {
+						ID: 2916285,
+						visible: false,
+						name: 'WordPress.com Way Better Example Blog',
+						URL: 'https://example2.wordpress.com',
+						options: {
+							unmapped_url: 'https://example2.wordpress.com'
+						}
+					}
+				}
+			}
+		};
+		const sites = getSites( state );
+		expect( sites ).to.eql( [
```

Comment:
To be honest, I was copying off of `getVisibleSites`, thinking it might be the thing to follow. I would (and now will) do the following:
- check for `length === 2`
- check both items for the right titles